distrho/src/DistrhoPluginLV2export.cpp

```diff
@@ -254,6 +255,13 @@ void lv2_generate_ttl(const char* const basename)
 #endif
         pluginString += ";\n\n";
 
+        // supportedOptions
+        pluginString += "    opts:supportedOption <" LV2_BUF_SIZE__nominalBlockLength "> ";
```

Comment:
these lines don't follow the rest of the syntax on this file.
leave the `,\n` at the end.